api/CustomElementRegistry.json

```diff
@@ -0,0 +1,65 @@
+{
+  "api": {
+    "CustomElementRegistry": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/CustomElementRegistry",
+        "description": "The CustomElementRegistry interface provides methods for registering custom elements and querying registered elements.",
+        "support": {
+          "chrome": {
+            "version_added": "33",
+            "notes": "Chrome 36+/Opera 20+ implemented a previous version of Custom Elements (v0) that used `.registerElement()`"
```

Comment:
Opera 20+ has nothing to do here, it is an entry about Chrome.
Chrome 36+ is unclear, we write it clear "From Chrome 36, …" (formulation can be done a bit differently).

Also it looks like the note is a restriction/limitation that has been added after the initial release (33). Are you sure of this?

Reply:
Yes a bit confusing going back and forth between caniuse. I feel there are some misleading statements. Perhaps need to address there. Looks like 54 is v1 true version.

Fixed in https://github.com/mdn/browser-compat-data/pull/623/commits/85a11a308317ec23490c4dfb801535ff2e372458